unity-client/Assets/Batch/AssetBundleMenuItems.cs

```diff
@@ -55,6 +57,15 @@ static void DumpAreaToMax(AssetBundleBuilder builder, int x, int y)
             builder.DumpArea(new Vector2Int(x, y), new Vector2Int(10, 10), (error) => DumpAreaToMax(builder, nextX, nextY));
         }
 
+        [MenuItem("Decentraland/Asset Bundle Builder/Dump Tomb Chaser")]
```

Comment:
I guess this is debug code to be able to quickly check that the fix worked with the affected scene. Is there any reason to keep it?

Reply:
If you look at this file, there are several similar menu items that dump several locations. I use them from time to time for testing and deleting it'd imply having to rewrite it each time I want to test this scene.

A better approach'd be to build a custom editor window for this purpose, but for now this is the only way I have to test important scenes.